examples/server/webui/src/main.js

```diff
@@ -33,7 +36,7 @@ const CONFIG_DEFAULT = {
   systemMessage: 'You are a helpful assistant.',
   showTokensPerSecond: false,
   // make sure these default values are in sync with `common.h`
-  samplers: 'edkypmxt',
+  samplers: 'dkypmxt',
```

Comment:
This should not be changed. With #10803, the penalties sampler has to be manually inserted, instead of always being present at the start.

Reply:
Ah yeah, I took the incorrect base branch. Fixed now, thanks!